### PR TITLE
Display a customisable message at the top of the report (e.g. link to policy document)

### PR DESCRIPTION
**Pull request by HuongNV13**
Hi @timhunt ,

Please help me to review it

The code was tested done by our tester

Thanks,

### CODE REVIEW

**Comment by timhunt** [comment]
Tl;dr; It is spot on. Thanks.


There was one thing I was worried about:

1. Since the config setting is user-input HTML, should it be passed thorugh format_text on output?

I tried to examples in core Moodle code, but just ended up getting totally confused:

* maintenance_message is a special case. The maintenance message can be displayed when Moodle is in mid upgrade, so you can't rely on anything working, so it is just output plain.
* backup/backup_async_message goes into an email, not displayed on-screen, so that is not the same.
* auth_instructions - if you decode auth/classes/output/login.php, this eventually gets passed through external_format_text (with hard-coded FORMAT_MOODLE). Since we are not doing anything related to web services, we should not copy that.
* auth_shibboleth does format_text($config->auth_instructions); but that turns out to be a setting that came from an admin_setting_configtextarea.

My guess is that you should do format_text(..., FORMAT_HTML).

However, looking through all the examples in the OU code, we just use the value directly. So I guess your code is consistent with all our other examples, and we should stick with that. Therefore, I will merge this pull request.
